### DISCUSSION TITLE
Preview environment seems not to update state from submitted to published

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#644] [Q&A] [answered]
Preview environment seems not to update state from submitted to published. 
Normally the state of a publication would be updated around 3 pm from submitted to published.
But this seems not to work anymore since the beginning of august.
Is there a reason for that?
It would help us that the status are update to published so we can test a complete workflow.

Regads,
Peter van de Kletersteeg

**Comment by pcolast** [comment, marked as answer]
Dear Peter van de Kletersteeg,

Unfortunately, there is an error in the export process of Preview and notices submitted as of 9 August 2023 do not acquire status “published”. We are working to fix this “[known issue](https://docs.ted.europa.eu/home/eforms/preview/index.html#_enotices2:~:text=Notices%20submitted%20to%20Preview%20eNotices2%20as%20of%209%20August%202023%20do%20not%20acquire%20status%20%E2%80%9Cpublished%E2%80%9D%20due%20to%20an%20error%20in%20the%20export%20process.%20We%20are%20working%20to%20fix%20this%20known%20issue%20as%20soon%20as%20possible.)” as soon as possible.

If you would like in the meantime to urgently test the workflow of a notice, let us know the business ID of the notice via a Helpdesk request.

**Comment by ghost** [comment]
Hi,
when will the publication on the preview-environment be available again? 
This problem is a serious obctacle when testing the publication on TED.

**Reply by AnnaKSZ** [reply]
Hi, 
I agree, this a blocker problem of our acceptance tests.

**Comment by pcolast** [comment]
Dear users,

We are working on the fix and we are trying to provide it as soon as possible.
We apologize for the inconvenience.

Thank you for your patience and understanding.

Have a nice day,
Stefan

**Comment by pcolast** [comment]
Dear users,

The issue has been fixed.

Have a nice day,
Stefan

**Comment by mgakt** [comment]
Dear Developers, Dear OP,

As far as we can see on the preview environment, the issue is appeared again. Since the 7th of September, no notice was published(We tried to publish noteice nine times.). Can you please look into it? If it's needed, I can provide various notice IDs.
Looking for your Kind response, Gabor

**Reply by pcolast** [reply]
Dear users,

Please be informed that we currently have an issue with the publication process in Preview.
We are working on the fix and we are trying to provide it as soon as possible.
We apologize for the inconvenience.

Thank you for your patience and understanding.

Have a nice day,
Stefan

**Reply by mgakt** [reply]
Dear Developer, Dear OP and Dear Stefan,

We managed to publish some notices yesterday. Is it a permanent fix, or did we just have a fortune? Please give a follow-up on this question!

We are looking for your kind response!
Gábor

**Reply by pcolast** [reply]
Dear users, Dear Gábor,

The root cause of the current issue has been identified, it is a new one. For the time being we don't have a new version of the application with the fix in the Preview. Therefore, longer delays with status change can still occur from time to time. We are trying to sort-out the submitted notices with workarounds.

We apologize for the inconvenience.

Thank you.

Have a nice day,
Stefan
